recipes/default.rb

```diff
@@ -3,6 +3,8 @@
 # Recipe:: default
 #
 
-%w( repo install configure service ).each do |r|
+Chef::Recipe, Chef::Resource].each { |l| l.send :include, ::Extensions }
```

Comment:
i think this is a syntax error? (missing opening bracket on the array), and I think we'd want this to be at the bottom of the module file so it's mixed in before chef evaluates the recipes, and so cookbook consumers can benefit from the library methods without having to have the default recipe in their run-list (e.g. if for some reason the service or config recipe doesn't work for them).